coldfront/core/allocation/views_/secure_dir_views.py

```diff
@@ -982,6 +985,12 @@ def show_rdm_consultation_form_condition(self):
         cleaned_data = self.get_cleaned_data_for_step(step) or {}
         return cleaned_data.get('rdm_consultation', False)
 
+    def __get_department(self, form_data):
+        """Return the department that the user submitted."""
+        step_number = self.step_numbers_by_form_name['ata_description']
```

Comment:
Received this error when submitting a new secure directory request.

```
[2023-10-27 17:41:00,355] ERROR secure_dir_views.done: 'ata_description'
Traceback (most recent call last):
  File "/var/www/coldfront_app/coldfront/coldfront/core/allocation/views_/secure_dir_views.py", line 911, in done
    department = self.__get_department(form_data)
  File "/var/www/coldfront_app/coldfront/coldfront/core/allocation/views_/secure_dir_views.py", line 990, in __get_department
    step_number = self.step_numbers_by_form_name['ata_description']
KeyError: 'ata_description'
```

Even after fixing the typo, is this the right key?

Reply:
typo, fixed it and seems to work